pkg/metastore/hive/ms_client.go

```diff
@@ -76,22 +111,27 @@ func (c *MSClient) Close() error {
 }
 
 func (c *MSClient) CopyOrMerge(fromDB, fromTable, toDB, toTable, toBranch, serde string, partition []string) error {
+	return c.CopyOrMergeToClient(fromDB, fromTable, toDB, toTable, toBranch, serde, partition, *c)
+}
+
+func (c *MSClient) CopyOrMergeToClient(fromDB, fromTable, toDB, toTable, toBranch, serde string, partition []string, toClient MSClient) error {
```

Comment:
Consider calling the method - CopyOrMergeTo.
Without the Client, it is enough to understand that the operation accepts the target/destination.

Reply:
👍